includes/admin/reporting/tools/class-give-tools-delete-test-transactions.php

```diff
@@ -164,10 +164,9 @@ public function process_step() {
 
 			return true;
 		} else {
-			update_option( 'give_earnings_total', 0 );
-			delete_transient( 'give_earnings_total' );
-			delete_transient( 'give_estimated_monthly_stats' . true );
-			delete_transient( 'give_estimated_monthly_stats' . false );
+			update_option( 'give_earnings_total', give_get_total_earnings( true ) );
```

Comment:
@DevinWalker I updated code to set `give_earnings_total` to latest income total instead `0`